src/docfx/build/Build.cs

```diff
@@ -29,52 +29,47 @@ private static async Task<bool> BuildDocset(string docsetPath, string outputPath
         {
             List<Error> errors;
             Config config = null;
-            PackageResolver packageResolver = null;
 
-            using (var errorLog = new ErrorLog(docsetPath, outputPath, () => config, options.Legacy))
-            {
-                var stopwatch = Stopwatch.StartNew();
-
-                try
-                {
-                    // load and trace entry repository
-                    var repository = Repository.Create(docsetPath);
-                    Telemetry.SetRepository(repository?.Remote, repository?.Branch);
-                    var locale = LocalizationUtility.GetLocale(repository, options);
+            using var errorLog = new ErrorLog(docsetPath, outputPath, () => config, options.Legacy);
```

Comment:
would be nice if you add a list of new language syntax changed in this PR in the description.

Reply:
updated description